pwa/components/admin/repairers/RepairersList.tsx

```diff
@@ -63,6 +63,12 @@ export const RepairersList = (): JSX.Element => {
     fetchRepairers();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    if (searchTerm.length >= 2) {
+      fetchRepairers();
+    }
+  }, [searchTerm]);
+
```

Comment:
```suggestion
  useEffect(() => {
    if (searchTerm.length === 0 || searchTerm.length >= 2) {
      fetchRepairers();
    }
  }, [fetchRepairers, searchTerm, currentPage]);

```
This check `searchTerm.length === 0` is when you delete all the chars from the input, the app need to displays all results again,
Wrap the `fetchRepairers` function in a `useCallback` to memoize it and avoid multiple rerender.
Then put the `currentPage` into the deps array, so when the pagination is active, this useEffect will be called